lib/core/src/connection_pool.cpp

```diff
@@ -195,7 +234,9 @@ namespace irods
                                             const std::function<void()>& _on_login_error)
     {
         auto& ctx = conn_ctxs_[_index];
-        ctx.creation_time = std::time(nullptr);
+
+        ctx.creation_time = std::chrono::steady_clock::now();
```

Comment:
No point in doing this if `number_of_seconds_before_connection_refresh` isn't set.

Reply:
Done